### DISCUSSION TITLE
cm311-1a s905l3a电视盒子，已经刷了armbian到emmc，现在无法从U盘里的coreelec/emuelec启动了

### DISCUSSION THREAD

**Discussion by sddzcuigc** [ophub/amlogic-s9xxx-armbian#1752] [Amlogic]
怎样修改U-boot启动顺序，从U盘启动？

**Comment by ophub** [comment]
这两个系统互斥，盒子里是ce时也无法启动armbian系统，只有安卓系统/armbian/openwrt可以emmc和usb互相支持启动。

想使用ce就线刷安卓系统恢复原版盒子即可。

**Comment by sddzcuigc** [comment]
原来是这个样子。另外我下载了armbian-software里的desktop，但是蓝牙就算执行了bt-install在设置里面，仍然显示 No bluetooth Found。这个系统的蓝牙是没有驱动吗

**Comment by sddzcuigc** [comment]
可以了，原来是用错了dtb | [xcray](https://github.com/xcray) | 32. [2022-09-16] After his test feedback in [Issues](https://github.com/ophub/amlogic-s9xxx-armbian/issues/557), Added `Bluetooth` support for `CM311-1a-YST (s905l3a)`, `enhanced` wired network stability, and added dedicated `meson-g12a-s905l3a-cm311.dtb` file. |
